manifests/r/Rapid7/Metasploit/6.1.44.1/Rapid7.Metasploit.locale.en-US.yaml

```
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Rapid7.Metasploit
+PackageVersion: 6.1.44.1
+PackageLocale: en-US
+Publisher: Rapid7, Inc.
+PublisherUrl: https://www.rapid7.com/
+PublisherSupportUrl: https://www.rapid7.com/for-customers/
+PrivacyUrl: https://www.rapid7.com/privacy-policy/
+Author: Rapid7, Inc.
+PackageName: Metasploit Framework

```

```suggestion
PackageName: Metasploit-framework
```
This must match Control Panel.